scrapi/base/helpers.py

```diff
@@ -135,7 +136,10 @@ def oai_process_uris(*args):
                 provider_uris.append(found_url)
 
     try:
-        canonical_uri = (provider_uris + object_uris)[0]
+        if use_doi and len(object_uris) > 0:
+            canonical_uri = object_uris[0]
```

Comment:
hmm, I guess there is no way to split out DOIs pointing to splash pages vs elsewhere. Though if DOIs usually point to splash pages, should they be in providerUris instead?